README.md

```diff
@@ -26,45 +26,6 @@ The cluster definition file enables the following customizations to your Docker
 * [Managed Disks](examples/disks-managed) (under private preview) - shows how to use managed disks 
 * [Large Clusters](examples/largeclusters) - shows how to create cluster sizes of up to 1200 nodes
 
-## Development (Docker)
```

Comment:
I thought this was linked in a few more places, but I think you found the only link in you change below so this is good.